checkstyle.xml

```diff
@@ -25,6 +25,10 @@
     <module name="FileTabCharacter">
         <property name="eachLine" value="true"/>
     </module>
+    <module name="LineLength">
+        <property name="max" value="120"/>
```

Comment:
let's fix the whitespace on these 3 lines then yea let's :shipit:

Reply:
how's that? got some weird inconsistency between tab length in vim vs github

Reply:
Sounds good